src/man/sssd-ipa.5.xml

```diff
@@ -536,6 +536,18 @@
                             from trusted domains directly while on a client
                             it will ask an IPA server.
                         </para>
+                        <para>
+                            NOTE: There are currently some assumptions that
+                            must be met when SSSD is running on an IPA server.
+                            <itemizedlist>
+                                <listitem>
+                                    <para>The <quote>ipa_server</quote> option must be configured to point to the IPA server itself</para>
```

Comment:
Is this the default we have? Just wondering whether we should enforce this in the code as well (not for this ticket though)

Reply:
After discussing with @jhrozek on IRC, I'd like to suggest to explicit mention that this value is automatically set up by tools (like the IPA installer).

Reply:
Fixed